dbms/src/Flash/Coprocessor/TiDBTableScan.h

```diff
@@ -50,6 +48,10 @@ class TiDBTableScan
     {
         return table_scan->executor_id();
     }
+    const tipb::Executor * getTableScan() const
```

Comment:
It seems that this method can be avoided

Reply:
Only have to implement genSchema and genNamesAndTypes functions with the TiDBTableScan struct as the input? I will fix it.

Reply:
yes